### PR TITLE
sample profile route

### PR DESCRIPTION
**Pull request by m-dabas**
This is a sample /profile route where you can change your information.

### CODE REVIEW

File: app/controllers/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+class ProfileController extends \BaseController {
+
+    public function showProfileView() {
+        if((Auth::check()))
+            return View::make('profile');
+
+        return View::make('login');
```
Comment (PaulSzyller) [inline app/controllers/ProfileController.php]: Don't make a login view here, redirect to /login route instead.
`return Redirect::to('/login')`

**Comment by PaulSzyller** [comment]
Instead of a Change FIrst Name form like this, I think it'd be better to display the information and have an edit button next to it.

First just focus on displaying all the profile information there in a decent way.
After that you can add an edit button (just for first name or whatever at first).

Also you can start by just having a text input right next to the field and a submit button and if it works and you're comfortable with it, try to make it hidden until someone clicks on edit.

**Comment by PaulSzyller** [comment]
After talking with Kurt, maybe it'd be better to have it this way:
-first focus on displaying everyting about the user without worrying about editing
-then we can add an edit button that pops up a modal to edit everything at once

**Comment by m-dabas** [comment]
There is now a card for each item and the Account button in the top right is now a functional Profile button.

screenshot from 2016-08-07 14 37 59

In addition to removing password from the display (but not the edit form), I think the Security Q&A should be removed as well.

**Comment by PaulSzyller** [comment]
I like the changes to the navbar and controller.
For the profile view itself, maybe put everything in 1 card instead of a card for each attribute.

**Comment by m-dabas** [comment]
How's this?
I put everything in a single card and separated each item with a horizontal rule.

**Comment by PaulSzyller** [comment]
Looks good.

**Comment by PaulSzyller** [comment]
Once we agree on the final look (I think we're pretty close).
Do you want to merge it or do you want to work on the edit modal in the same branch?

**Comment by m-dabas** [comment]
I think it's better to make the edit modal in a different branch.

**Comment by m-dabas** [comment]
After making the headers smaller and splitting it into two cards, everything fits nicely on my screen.

**Pull request merged**